File: java/sourceafis/src/sourceafis/matching/minutia/MinutiaPair.java
package sourceafis.matching.minutia;
 
public class MinutiaPair{
    public int probe;
    public int candidate;
    /*
     * Empty Constcture 
     */
    public MinutiaPair()
    {
    }
    public MinutiaPair(int probe, int candidate)
    {
        this.probe = probe;
        this.candidate = candidate;
    }
    
    public MinutiaPair clone() {
    	return new MinutiaPair(this.probe,this.candidate);   		 
    }
}

File: SourceAFIS/Matching/MinutiaPair.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace SourceAFIS.Matching
{
    public struct MinutiaPair
    {
        public int Probe;
        public int Candidate;

        public MinutiaPair(int probe, int candidate)
        {
            Probe = probe;
            Candidate = candidate;
        }
    }
}
